dscreator/sources/ferrybox/extractor.py

```diff
@@ -93,7 +93,7 @@ def first_timestamp(self) -> datetime:
         Padded with 10 sec
         """
```

Comment:
why first timestamp or that date?

Reply:
was a random date to get started but should be updated